### ISSUE TITLE
PyTorch GEMM/MLP into benchmarks

### ISSUE THREAD

**Issue by rengolin** [plaidml/tpp-mlir#776]
Now that we can generate models with PyTorch into MLIR linalg and can consume them in our compiler, we need to add them to the benchmark list. Once we fix #773, we'll hopefully get the same performance as the ones from `mlir-gen`.

To avoid compatibility issues and creating too many scripts, we should just have two models in https://github.com/plaidml/mlir-generator to create similar benchmarks as we have ({GEMM only / MLP }, 3-layers, 256 batch, 1024 layer size, no softmax, {FP32 / BF16}).

The 4 new models should be added to the "gemm_models" and "mlp_models" (as in #774) two each (32, 16).

This should conclude the task that connects PyTorch (Dynamo) to our compiler. Of course, not all models work, and the connection is flaky (manual steps), but automation of that particular task and adding new models and making them work will be handled elsewhere.
